src/Migration/Sources/Appwrite.php

```diff
@@ -977,15 +978,24 @@ private function exportFiles(int $batchSize)
                 );
 
                 foreach ($response['files'] as $file) {
-                    $this->exportFileData(new File(
-                        $file['$id'],
-                        $bucket,
-                        $file['name'],
-                        $file['signature'],
-                        $file['mimeType'],
-                        $file['$permissions'],
-                        $file['sizeOriginal'],
-                    ));
+                    try {
+                        $this->exportFileData(new File(
+                            $file['$id'],
+                            $bucket,
+                            $file['name'],
+                            $file['signature'],
+                            $file['mimeType'],
+                            $file['$permissions'],
+                            $file['sizeOriginal'],
+                        ));
+                    } catch (\Exception $e) {
```

Comment:
```suggestion
                    } catch (\Throwable $th) {
```

Make sure to catch Throwable everywhere. And then create your own exception out of it